### ISSUE TITLE
how to generate a daily note as YYYY_MM_DD.md

### ISSUE THREAD

**Issue by Dasein1998** [epwalsh/obsidian.nvim#85]
### 🚀 The feature, motivation and pitch

when the [obsidian.nvim](https://github.com/epwalsh/obsidian.nvim) generate the daily notes, it create a file which format is YYYY-MM-DD, and with some frontmatter.
How would I create a daily note as YYYY_MM_DD.md ? and with a custum content?

**Comment by epwalsh** [comment]
Hey @Dasein1998, this isn't possible at the moment but it wouldn't be hard to add. We'd just need to add another configuration option [here](https://github.com/epwalsh/obsidian.nvim/blob/a27b8342c145f0135f432ebfb45cfae60bf0d121/lua/obsidian/config.lua#L58-L68) and then use that option [here](https://github.com/epwalsh/obsidian.nvim/blob/a27b8342c145f0135f432ebfb45cfae60bf0d121/lua/obsidian/init.lua#L168).

**Comment by jaymorelli96** [comment]
Hi @epwalsh  can I pick this up? I am just starting on the open source world and I have been using this plugin for a bit and is exactly what I was looking for and would love to contribute more for this project. I may need more guidance, would be possible for me to reach it out to you in case I get stuck or something? TIA

**Comment by epwalsh** [comment]
Hey @jaymorelli96, sorry I forgot to link this PR before, but someone is already working on it: https://github.com/epwalsh/obsidian.nvim/pull/49

**Comment by jaymorelli96** [comment]
No problem @epwalsh I will keep my on for some good first issue 😄 Thanks for the amazing plugin

**Comment by evtuhovich** [comment]
Fixed in #138, could you check please?

**Comment by NachoxMacho** [comment]
Not the original poster but I was able to accomplish this with the following config section for anyone stumbling across this issue.
```
    daily_notes = {
        date_format = "%Y_%m_%d",
    },
```
